### PR TITLE
MGMT-18905: move Terraform params to env

### PR DESCRIPTION
**Pull request by eifrach**
- Change nutanix Terraform to use ENV params instead of  exporting to file.

Add the ability to add new variables via vault 

TODO: remove old hcl value

### CODE REVIEW

File: ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh [hunk 1]
```diff
@@ -33,6 +33,18 @@ iso_name = "assisted-test-infra-machine-template"
 build_id = "${BUILD_ID}"
 EOF
 
+# Terraform Params
+# move to new ENV value with auto load
+export CI_CREDENTIALS_DIR=/var/run/vault/assisted-ci-vault
+
+for file in $CI_CREDENTIALS_DIR/TF_VAR_*; do
+   if [[  "$file" == *"TF_VAR_"* ]]; then
+    key=$(basename -- $file)
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh:42]:
```suggestion
     key=$(basename -- $file)
```

**Review by adriengentil — APPROVED** [review]

File: ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh (hunk 1 above)
Comment (eifrach) [reply ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh]: done

File: ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh [hunk 2]
```diff
@@ -33,6 +33,18 @@ iso_name = "assisted-test-infra-machine-template"
 build_id = "${BUILD_ID}"
 EOF
 
+# Terraform Params
+# move to new ENV value with auto load
+export CI_CREDENTIALS_DIR=/var/run/vault/assisted-ci-vault
+
+for file in $CI_CREDENTIALS_DIR/TF_VAR_*; do
+   if [[  "$file" == *"TF_VAR_"* ]]; then
```
Comment (danmanor) [inline ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh:41]: why both `if` and `/TF_VAR_*` ?

Comment (eifrach) [reply ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh:41]: yes you right, This was for previous version  - i'll  fix it

Comment (danmanor) [reply ci-operator/step-registry/assisted/nutanix/setup/machine/assisted-nutanix-setup-machine-commands.sh:41]: Thanks !

**Pull request merged**